src/main/webapp/app/entities/course/course.component.html

```diff
@@ -21,8 +21,8 @@ <h4 id="course-page-heading" jhiTranslate="artemisApp.course.home.title">Courses
         <table class="table table-striped">
             <thead>
                 <tr jhiSort [(predicate)]="predicate" [(ascending)]="reverse" [callback]="callback">
-                    <th></th>
-                    <th jhiSortBy="id">
+                    <th class="d-none d-md-table-cell"></th>
+                    <th class="d-none d-md-table-cell" jhiSortBy="id">
```

Comment:
the user should be able to navigate into the view page

Reply:
I checked, and the `View` button is still available, so they can still navigate to this page :+1: